### PR TITLE
[ZEPPELIN-5338]. Add logging prefix in interpreter.sh for easy diagnose

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?

Currently, we have to change log4j.properties to get more output from script `interpreter.sh` in log4j output for diagnose. This PR is to add one trick: as long as the output is start with `[INFO]` in `interpreter.sh`, it will be redirected to log4j output.  

### What type of PR is it?
[ Improvement ]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-5338

### How should this be tested?
* CI pass

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: bin/interpreter.sh [hunk 1]
```diff
@@ -296,5 +296,6 @@ else
 fi
 
 # Don't remove this echo, it is for diagnose, this line of output will be redirected to java log4j output.
-echo "Interpreter launch command: ${INTERPRETER_RUN_COMMAND[@]}"
+# Output that starts with `[INFO]` will be redirected to log4j output.
```
Comment (cuspymd) [inline bin/interpreter.sh:299]: Isn't it better to express "log4j INFO output" or "log4j INFO message" more precisely than "log4j output"? Depending on the log level setting, debug messages can also be displayed in the output.

Comment (zjffdu) [reply bin/interpreter.sh]: Thanks for the comment @cuspymd It is fixed now.

**Comment by zjffdu** [comment]
@cuspymd Any more comment ?

**Review by cuspymd — APPROVED** [review]
Looks good!!

**Comment by zjffdu** [comment]
Will merge if no more comment

**Pull request closed without merging**